docs/resources/fivepunch-cops-and-robbers/index.md

```diff
@@ -0,0 +1,508 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# fivepunch-cops-and-robbers
+
+---
+
+**A tactical team-based shooter for Fivem**
+
+![Cops and Robberts](fivepunch-cops-and-robbers-banner.png)
+
+---
+
+## Installation
+
+### Download
+
+After the purchase you will receive an email from Tebex containing the instructions to download the resource.
+
+Alternatively, you can go to https://keymaster.fivem.net/asset-grants and download it.
+
+### Setting up
+
+1. Extract the `fivepunch-cops-and-robbers.pack.zip` file to a folder called `fivepunch-cops-and-robbers` inside your resources folder.
+
+2. Add the following to your `server.cfg` to start the resource.
+
+```cfg
+ensure fivepunch-cops-and-robbers
+```
+
+---
+
+## Configuration
+
+You can modify all configurations about minigame and matchmaking in file `config.lua`. But don't worry, you can safely use all default options.
```

Comment:
All the configurations for the mini-game and the matchmaking are in the file `config.lua`. Tweak it to best suit your needs.